### PR TITLE
Add option to remove toggle from button card

### PR DESCRIPTION
**Pull request by kdkavanagh**
Also ensures that the global font family is used in button card

### CODE REVIEW

File: packages/components/src/Cards/ButtonCard/index.tsx [hunk 1]
```diff
@@ -138,6 +138,7 @@ const Footer = styled.div`
 const Title = styled.div`
   color: var(--ha-S500-contrast);
   font-size: 0.7rem;
+  font-family: var(--ha-font-family);
```
Comment (shannonhochkins) [inline packages/components/src/Cards/ButtonCard/index.tsx:141]: This isn't necessary - it'll inherit from the body by default

File: packages/components/src/Cards/ButtonCard/index.tsx [hunk 2]
```diff
@@ -146,9 +147,21 @@ const Title = styled.div`
...
+  ${(props) =>
+    `color: ${props.textColor ? props.textColor : "var(--ha-S50-contrast)"};
+  `}
+  ${(props) =>
+    `font-size: ${props.textSize ? props.textSize : "0.8rem"};
+  `}
+  ${(props) =>
+    `text-align: ${props.textAlign ? props.textAlign : "center"};
+  `}
+  font-family: var(--ha-font-family);
```
Comment (shannonhochkins) [inline packages/components/src/Cards/ButtonCard/index.tsx:164]: These styles won't ever work as you're not passing through these props from the parent component.

Comment (shannonhochkins) [reply packages/components/src/Cards/ButtonCard/index.tsx:164]: Just an FYI as you might not be aware, but you can already achieve the above by adding cssStyles prop which is supported on pretty well every component. So can you please revert the above style changes?

```
<ButtonCard cssStyles={`
    .description {
        color: red;
    }
`} />

Comment (shannonhochkins) [reply packages/components/src/Cards/ButtonCard/index.tsx:164]: As mentioned above, please move this font-family declaration to the body targeting all elements

Comment (kdkavanagh) [reply packages/components/src/Cards/ButtonCard/index.tsx:164]: Ah yes, you're right.  I'm using these in my local fork to create a different button look:

image

**Comment by shannonhochkins** [comment]
Closed as I've made the same changes in another PR as youve not updated this recently

**Pull request closed without merging**